packages/training/book/src/development/codebase-map.md

```
@@ -0,0 +1,296 @@
+
+```
+TypeScript:
+  TrajectoryRecorder → PostgreSQL/JSON
+
+Python:
+  JsonTrajectoryReader ← PostgreSQL/JSON
+```
+
+### Shared Config
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)